src/sonic-bgpcfgd/bgpcfgd/managers_bbr.py

```diff
@@ -82,6 +94,23 @@ def __read_pgs(self):
                 res[pg_name] = pg_afs
         return res
 
+    def get_bbr_status_from_config_db(self):
+        """
+        Read BBR status from CONFIG_DB
+        :return: BBR status from CONFIG_DB or None if not found
+        """
+        config_db = swsscommon.ConfigDBConnector()
+        if config_db is None:
```

Comment:
if at init time config DB connection fails, we are reverting to default state from constants.yml. I suggest we throw a log here before returning None.

Reply:
fixed